content/en/docs/installing-on-linux/on-premises/install-kubesphere-on-vmware-vsphere.md

```diff
@@ -0,0 +1,401 @@
+    enabled: false
+    logsidecarReplicas: 2
+  metrics_server:                    # Whether to install metrics-server. IT enables HPA (Horizontal Pod Autoscaler).
+    enabled: true
+  monitoring:                        #
+    prometheusReplicas: 1            # Prometheus replicas are responsible for monitoring different segments of data source and provide high availability as well.
+    prometheusMemoryRequest: 400Mi   # Prometheus request memory
+    prometheusVolumeSize: 20Gi       # Prometheus PVC size
+    alertmanagerReplicas: 1          # AlertManager Replicas
+  multicluster:
+    clusterRole: none  # host | member | none  # You can install a solo cluster, or specify it as the role of host or member cluster
+  networkpolicy:       # Network policies allow network isolation within the same cluster, which means firewalls can be set up between certain instances (Pods).
+    enabled: false     
+  notification:        # It supports notification management in multi-tenant Kubernetes clusters. It allows you to set AlertManager as its sender, and receivers include Email, Wechat Work, and Slack.
+    enabled: false
+  openpitrix:          # Whether to install KubeSphere Application Store. It provides an application store for Helm-based applications, and offer application lifecycle management
+    enabled: false
+  servicemesh:         # Whether to install KubeSphere Service Mesh (Istio-based). It provides fine-grained traffic management, observability and tracing, and offer visualization for traffic topology
+    enabled: false
+```
+#### 1.2  Verify the multi-node installation
+
+If `Welcome to KubeSphere!` it returned after `./kk create cluster` process completed, then congratulation! you are ready to go.
+
+```bash
+**************************************************
+#####################################################
+###              Welcome to KubeSphere!           ###
+#####################################################
+Console: http://10.10.71.214:30880
+Account: admin
+Password: P@88w0rd
+NOTES：
+  1. After logging into the console, please check the
+     monitoring status of service components in
+     the "Cluster Management". If any service is not
+     ready, please wait patiently until all components 
+     are ready.
+  2. Please modify the default password after login.
+#####################################################
+https://kubesphere.io             2020-08-15 23:32:12
+#####################################################
+```
+
+#### 1.3  log in the console
+
+You will be able to use default account and password `admin / P@88w0rd` to log in the console `http://{$IP}:30880` to take a tour of KubeSphere. Please change the default password after logging in.
+
+![](/images/docs/vsphere/login.png)
```

Comment:
Add a single section to add the description for **Enable Pluggable Plugins**. see [this guide](https://v3-0.docs.kubesphere.io/docs/installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-gke/) for details.